src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -173,7 +173,7 @@ const DiscoverySearch = ({
               <MemoIcArrowBackWhite
                 width={20}
                 height={12}
-                fill={COLORS.black}
+                fill={COLORS.white2}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color of the `MemoIcArrowBackWhite` component has been changed from black to white. Ensure that this change doesn't affect the visibility of the icon, especially if the background is also white or a light color.
```diff
- fill={COLORS.black}
+ fill={COLORS.white2}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->